### PR TITLE
Updates patternfly dependencies

### LINKED ISSUE #11105: Update Patternfly
##### ISSUE TYPE
 - Feature Idea

##### SUMMARY
Last time we updated Patternfly was about 4 months ago.

### PR DESCRIPTION
**Pull request by AlexSCorey**
##### SUMMARY
This resolves #11105
##### ISSUE TYPE
-dependency upgrade

##### COMPONENT NAME
 - UI

### CODE REVIEW

**Comment by nixocio** [comment]
Upgrade of PF should NOT add this issue here - https://github.com/ansible/awx/issues/11224 - Verify that this issue does not exist with the newer version.
A previous version of PF did, that is why we needed to roll back a previous upgrade.
I triggered the e2e tests.

**Comment by nixocio** [comment]
@AlexSCorey this PR is changing the behavior of the drag/drop of elements.

On devel:

https://user-images.githubusercontent.com/9053044/144258863-98b0574b-d5d1-4409-bd6f-8d3d40608dec.mov

On the code related to this PR.

https://user-images.githubusercontent.com/9053044/144257751-61a0a724-6e47-472f-b7e3-42815f71b7ee.mov

Do we need to upgrade those components? I was able to run the code on devel on the PF version of this branch.

All tests done on Firefox.

**Review by nixocio — CHANGES_REQUESTED** [review]
The drag/drop behavior is different from devel.

File: awx/ui/src/components/SelectedList/DraggableSelectedList.js [hunk 1]
```diff
@@ -21,114 +17,94 @@ const RemoveActionSection = styled(DataListAction)`
...
+                <DragWrapper disabled>
+                  <GripVerticalIcon />
+                </DragWrapper>
+                <span id={rowPosition}>{`${rowPosition}. ${label}`}</span>
+                <RemoveActionSection>
+                  <Button
+                    onClick={() => removeItem(label)}
+                    variant="plain"
+                    aria-label={t`Remove`}
+                    ouiaId={`draggable-list-remove-${label}`}
```
Comment (nixocio) [inline awx/ui/src/components/SelectedList/DraggableSelectedList.js]: removal of `data-cy` may break tests.

File: awx/ui/src/components/SelectedList/DraggableSelectedList.js [hunk 2]
```diff
@@ -21,114 +17,94 @@ const RemoveActionSection = styled(DataListAction)`
...
-                />
-                <RemoveActionSection aria-label={t`Actions`} id={rowPosition}>
+      <DragDrop onDrop={dragItem}>
+        <Droppable>
+          {selected.map(({ name: label, id }, index) => {
+            const rowPosition = index + 1;
+            return selected.length < 2 ? (
+              <div key={rowPosition}>
+                <DragWrapper disabled>
+                  <GripVerticalIcon />
```
Comment (nixocio) [inline awx/ui/src/components/SelectedList/DraggableSelectedList.js]: @AlexSCorey overall good job, this new implementation is leaner.

However I noticed a few points:

On the previous implementation once the user hover over this button - we allow the user to drag the row.

There were lines between rows, and the `x` to remove that row was positioned on the end of the row. This change may break visual tests..

Current implementation:

image

Current devel implementation.

image

**Review by nixocio — CHANGES_REQUESTED** [review]

**Comment by akus062381** [comment]
I'm going to run a Jenkins run against this branch

**Review by nixocio — APPROVED** [review]

File: awx/ui/src/setupTests.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { configure } from 'enzyme';
-import Adapter from 'enzyme-adapter-react-16';
+import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
```
Comment (nixocio) [inline awx/ui/src/setupTests.js]: @AlexSCorey this line is failing the linter, and tests. Other than that changes look good.

**Review by nixocio — CHANGES_REQUESTED** [review]

**Comment by akus062381** [comment]
@AlexSCorey please check with me before merging this. As the run above shows, we've got 16 failures on this branch.

**Review by akus062381 — APPROVED** [review]

**Comment by AlexSCorey** [comment]
drag

**Review by nixocio — APPROVED** [review]
@AlexSCorey we need to file an issue to cover accessibility.

https://www.patternfly.org/v4/components/drag-and-drop

> Note: Keyboard accessibility and screen reader accessibility are still in development.

**Pull request merged**